### PR TITLE
Ignore keepalives before connection_ack in websockets transport

### PR DESCRIPTION
**Pull request by leszekhanusz**
This should fix bug #108

### CODE REVIEW

**Comment by hugbubby** [comment]
plz merg

File: gql/transport/websockets.py [hunk 1]
```diff
@@ -175,6 +175,23 @@ async def _receive(self) -> str:
...
+
+            answer_type, answer_id, execution_result = self._parse_answer(init_answer)
+
+            if answer_type == "connection_ack":
+                return
+
+            if answer_type != "ka":
+                raise TransportProtocolError(
+                    "Websocket server did not return a connection ack"
+                )
```
Comment (KingDarBoja) [inline gql/transport/websockets.py:193]: Shouldn't be `"Webscocket server did not return a keep-alive"` ?

Comment (leszekhanusz) [reply gql/transport/websockets.py:193]: No, we are waiting here for a connection_ack (just above)
If we don't receive a connection_ack, we check if it was instead a keep-alive.
If it is not a keep-alive here, it means that it is not a keep-alive and not a connection_ack so we raise the exception same as before.
If it is a keep-alive, then we continue in the loop and will listen to the next message received hoping to receive a connection_ack next.

Comment (KingDarBoja) [reply gql/transport/websockets.py:193]: Fair enough, got it wrong while reviewing it.

**Pull request merged**